drivers/sensor/meas/ms5837/ms5837.c

```diff
@@ -75,17 +75,17 @@ static void ms5837_compensate_30(const struct device *dev,
 
 	temp_sq = (int64_t)(data->temperature - 2000) * (data->temperature - 2000);
 	if (data->temperature < 2000) {
-		Ti = (3ll * dT * dT) / (1ll << 23);
-		OFFi = (3ll * temp_sq) / 1ll;
+		Ti = (3ll * dT * dT) / (1ll << 33);
+		OFFi = (3ll * temp_sq) / 1ll << 1;
```

Comment:
We need to add parenthesis here, otherwise division will take precedence to the left-shift operation.
```suggestion
		OFFi = (3ll * temp_sq) / (1ll << 1);
```

Reply:
Thanks, that is a great catch.
I am going to change this and submit again.
Question is: normally, what is the best practices to resubmit it?
The latest zephyr main moved MS5837 to new vendor folder.
Should I rebase on the latest zephyr main branch?

Reply:
First time to do the pull request and change, etc.
I changed the original branch according to ubieda's suggestion. 
It is great catch. Thanks.